core/src/test/java/org/apache/accumulo/core/file/blockfile/cache/BlockCacheManagerTest.java

```diff
@@ -27,6 +27,7 @@
 public class BlockCacheManagerTest {
 
   @Test
+  @SuppressWarnings("deprecation")
```

Comment:
What is being deprecated in this PR? This feature is being added as experimental. New experimental features don't typically cause new deprecations.

Reply:
`BlockCacheManager.getFullyQualifiedPropertyPrefix` methods are marked @Deprecated(since = "2.1.0")